src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -175,10 +198,57 @@ public bool PrimaryNodeIsRequired
         /// Service Id.
         /// </summary>
         public Guid ServiceId { get; set; }
+
         /// <summary>
         /// Deployment Id.
         /// </summary>
         public string DeploymentId { get; set; }
+
+        #region MultiClusterNetwork
+
+        /// <summary>
+        /// Whether this cluster is configured to be part of a multicluster network
+        /// </summary>
+        public bool HasMultiClusterNetwork
+        {
+            get
+            {
+                return !(string.IsNullOrEmpty(GlobalServiceId) || string.IsNullOrEmpty(ClusterId));
+            }
+        }
+
+        /// <summary>
+        /// Shared Global service Id between different datacenters/deployments.
+        /// </summary>
+        public string GlobalServiceId { get; set; }
+
+        /// <summary>
+        /// Cluster id (one per deployment, unique across all the deployments/clusters)
+        /// </summary>
+        public string ClusterId { get; set; }
+
+        /// <summary>
+        ///A list of cluster ids, to be used if no multicluster configuration is found in gossip channels.
+        /// </summary>
+        public IReadOnlyList<string> DefaultMultiCluster { get; set; }
```

Comment:
Why do we need `DefaultMultiCluster`? is this for fixed multi-cluster configurations?